### PR TITLE
Changes to Quick Start for clarity

### PR DESCRIPTION
**Pull request by dvanco**
Just wanted to do a smaller edit before I touch other English docs.  You've done a great job, but a little polish may make it better :)

### CODE REVIEW

File: documentation/content/manual/_index.en.md [hunk 1]
```diff
@@ -9,29 +9,37 @@ type: "docs"
...
-    * Or you define the parameters for the article on the BE overview page:
-        * Article Group: Moves the article to a group you specify.
-        * Item Maximum Bid: Defines the maximum bid that BE will submit for the item just before the auction ends.
-        * Item Auto-Bid Mode: Activates the auto-bid mode for the item. If it is inactive, BE will not automatically bid on this item.
+    * You can enter a maximum bid directly on the item page.
+      This will also "save" the item in the overview page - and it will remain there even if the item tab is closed.
+	* NOTE that the "Place bid" display on the item page will reflect the state of the "bid active" status of that item via an overlay from BE  
+{{< image src="/bid_overlay.en.png" alt="Overview Page/Tab" >}}	  
+    * Or you enter the bid for the item directly on the BE overview page:
+        * Group: By default, all items appear in "Group". Simply enter a new group name next to an item and it will move to a new group with the name you specify.
```
Comment (sweitzel) [inline documentation/content/manual/_index.en.md]: Shouldn't that be 
> Group: By default, all items appear in the group "Other Auctions". ...

File: documentation/content/manual/_index.en.md [hunk 2]
```diff
@@ -9,29 +9,37 @@ type: "docs"
...
+        * Your Maximum Bid: Defines the maximum bid that BE will submit for the item just before the auction ends.
+        * Item "Active" Mode: Activates the auto-bid mode for the item. If it is inactive, BE will not automatically bid on this item!
+	* NOTE that if you do not enter a maximum bid value on the BE overview page OR on the item page itself closing the item tab will remove it from the BE overview.
```
Comment (sweitzel) [inline documentation/content/manual/_index.en.md]: That not actually true, the item will only be removed, if the item is also not assigned to a custom group, when the tab is being closed.

**Review by sweitzel — CHANGES_REQUESTED** [review]
I think the changes are very good!
But we should correct the two topics which are feature-wise incorrect.

**Comment by dvanco** [comment]
OK - I've made the changes - I don't think I have to do another pull request, do I? My updates are there? (sorry - used to git from command line at work ;) )

**Review by sweitzel — APPROVED** [review]

**Pull request merged**